feeding/src/action/MoveAbove.cpp

```diff
@@ -54,8 +54,7 @@ bool moveAbove(
           planningTimeout,
           maxNumTrials,
           getConfigurationRanker(ada),
-          velocityLimits,
-          ::ada::TrajectoryPostprocessType::KUNZ);
+          velocityLimits);
```

Comment:
This is more of a question for y'all, but are you OK accepting whatever `libada` sets as the default type for the PP here? If so this is OK, but if you depend on the PP being `Kunz` it might be good to set that explicitly here.

Reply:
Right now, we're okay with whatever the default is. There's nothing special we need with Kunz.

Reply:
Groovy!